### PR TITLE
Let ores python files have more config paths

### PR DESCRIPTION
**Pull request by Ladsgroup**

### CODE REVIEW

**Comment by Ladsgroup** [comment]
We need this in prod

**Comment by Ladsgroup** [comment]
Also very tested :)

File: ores_celery.py [hunk 1]
```diff
@@ -1,4 +1,17 @@
...
+"""
+Run a celery worker node.
+
+:Usage:
+    ores_celery.py -h | --help
+    ores_celery.py [--config=<path>]...
+
+:Options:
+    -h --help        Prints this documentation
+    --config=<path>  The path to a yaml config file
```
Comment (akosiaris) [inline ores_celery.py]: I think you mean dir, not file

File: ores_wsgi.py [hunk 2]
```diff
@@ -1,4 +1,17 @@
...
+"""
+Run a uwsgi server.
+
+:Usage:
+    ores_wsgi.py -h | --help
+    ores_wsgi.py [--config=<path>]...
+
+:Options:
+    -h --help        Prints this documentation
+    --config=<path>  The path to a yaml config file
```
Comment (akosiaris) [inline ores_wsgi.py]: I think you mean directory

**Comment by halfak** [comment]
1. Why do we need this in prod?
2. We generally run this as a module via `uwsgi` or `celery worker`, so command line parameters aren't really an option. 

The only reason you can still run these as a script is for testing purposes.  See everything within the `if __name__ == '__main__':` block.

**Comment by akosiaris** [comment]
Cause the directory where the configuration normally resides is populated by scap. And scap, trying to be as atomic as possible, populates the entire directory under a revs/<git_hash_id> subdirectory and then just changes a symlink having the deploy/ directory point to the new /revs/git_hash_id. The result is the the puppet provisioned configuration file that has redis passwords and hosts, which is not possible to add in the deploy repo since it's public, does not exist after a scap deploy, causing ores to not start until puppet has ran.

We do need to add the functionality of allowing ORES to read an external to the repo directory, if the CLI params way will not work, we need something else.

**Pull request closed without merging**